buffer/src/main/java/io/netty/buffer/PooledSlicedByteBuf.java

```diff
@@ -114,17 +114,14 @@ public ByteBuf retainedSlice(int index, int length) {
     @Override
     public ByteBuf duplicate() {
         // Capacity is not allowed to change for a sliced ByteBuf, so length == capacity()
-        final ByteBuf duplicate = unwrap().slice(adjustment, capacity());
+        final ByteBuf duplicate = super.slice(adjustment, capacity());
```

Comment:
yes... Would be what happens when you construct  a new DuplicatedByteBuf from this one ?